### PR TITLE
theharvester 4.0.3

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

**Comment by chenrui333** [comment]
```
    error in beautifulsoup4 setup command: use_2to3 is invalid.
```

**Comment by iMichka** [comment]
Updating the resources should do it.

File: Formula/theharvester.rb [hunk 1]
```diff
@@ -26,6 +26,10 @@ class Theharvester < Formula
   uses_from_macos "libxml2"
   uses_from_macos "libxslt"
 
+  # How to update the resources
+  # tar -zxvf theHarvester-4.0.3.tar.gz
+  # cd theHarvester-4.0.3 && pip install -r requirements/base.txt
+  # run homebrew-pypi-poet on the freezed dependencies
```
Comment (SMillerDev) [inline Formula/theharvester.rb:32]: How do they suddenly become frozen dependencies here? Generally pip dependencies are not frozen.

Comment (chenrui333) [reply Formula/theharvester.rb:32]: yeah, I had to run the pip freeze in the virtualenv setup.

**Review by branchv — APPROVED** [review]

**Pull request closed without merging**